### PR TITLE
Tweaks some food descriptions/casing.

### PR DESCRIPTION
**Pull request by Shadeykins**
**What does this PR do:**
Polishes/modifies a few food item descriptions and standardizes casing across all food items. All food now has lowercase except in cases where a brand-name or a normally capitalized word is used (IE: Cuban carp). Fixes a couple strange grammatical constructions as well.

**Changelog:**
:cl:
tweak: Food descriptions/casing.
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/food/foods/sandwiches.dm [hunk 1]
```diff
@@ -97,39 +97,39 @@
...
 	desc = "This is absolutely Ei Nath."
 	icon_state = "spellburger"
 	filling_color = "#D505FF"
 	bitesize = 3
 	list_reagents = list("nutriment" = 6, "vitamin" = 1)
 
 /obj/item/reagent_containers/food/snacks/bigbiteburger
-	name = "Big Bite Burger"
-	desc = "Forget the Big Mac. THIS is the future!"
+	name = "big bite burger"
```
[the remark this answers is not in the document]
Comment (Shadeykins) [reply code/modules/food_and_drinks/food/foods/sandwiches.dm]: I've joined the words together to make the branding more obvious in that case. BigBite and SuperBite respectively.

File: code/modules/food_and_drinks/food/foods/soups.dm [hunk 2]
```diff
@@ -68,7 +68,7 @@
 	reagents.add_reagent("[extra_reagent]", 5)
 
 /obj/item/reagent_containers/food/snacks/wishsoup
-	name = "Wish Soup"
+	name = "wish soup"
```
[the remark this answers is not in the document]
Comment (Shadeykins) [reply code/modules/food_and_drinks/food/foods/soups.dm]: Is this genuinely a reference to an arcane and barely known Australian children's tale, or is that incidental? I am legitimately curious.

File: code/modules/food_and_drinks/food/foods/misc.dm [hunk 3]
```diff
@@ -58,7 +58,7 @@
 //////////////////////
 
 /obj/item/reagent_containers/food/snacks/aesirsalad
-	name = "Aesir salad"
+	name = "aesir salad"
```
[the remark this answers is not in the document]
Comment (Shadeykins) [reply code/modules/food_and_drinks/food/foods/misc.dm]: Fixed.

File: code/modules/food_and_drinks/food/foods/pasta.dm [hunk 4]
```diff
@@ -45,23 +45,23 @@
...
+	name = "spesslaw"
 	desc = "A lawyer's favourite."
 	icon = 'icons/obj/food/pasta.dmi'
 	icon_state = "spesslaw"
 	filling_color = "#DE4545"
 	list_reagents = list("nutriment" = 8, "synaptizine" = 10, "vitamin" = 6)
 
 /obj/item/reagent_containers/food/snacks/macncheese
-	name = "Macaroni cheese"
+	name = "macaroni cheese"
```
[the remark this answers is not in the document]
Comment (FalseIncarnate) [reply code/modules/food_and_drinks/food/foods/pasta.dm]: After confirming with maintainers, we want to use the American spellings whenever possible. This should be one of the following:
- macaroni and cheese
- macaroni & cheese
- mac'n'cheese

File: code/modules/food_and_drinks/food/foods/pizza.dm [hunk 5]
```diff
@@ -80,15 +80,15 @@
 	filling_color = "#e5b437"
 
 /obj/item/reagent_containers/food/snacks/sliceable/pizza/macpizza
-	name = "Macaroni cheese pizza"
+	name = "macaroni cheese pizza"
```
Comment (FalseIncarnate) [inline code/modules/food_and_drinks/food/foods/pizza.dm]: After confirming with maintainers, we want to use the American spellings whenever possible. This should be one of the following:
- macaroni and cheese
- macaroni & cheese
- mac'n'cheese

**Comment by Shadeykins** [comment]
> 
> 
> Almost there, just gotta Americanize some cheesy pasta.

Changed to the AmeriCanadian "mac n cheese" which is surprisingly rendered without apostrophes.

sorry Citined. :(

**Comment by Citinited** [comment]
@Shadeykins HOW DARE YOU 👿 👀

**Pull request merged**